src/Twig/Runtime/MagazineExtensionRuntime.php

```diff
@@ -5,13 +5,17 @@
 namespace App\Twig\Runtime;
 
 use App\Entity\Magazine;
+use App\Repository\MagazineSubscriptionRepository;
 use Symfony\Bundle\SecurityBundle\Security;
 use Twig\Extension\RuntimeExtensionInterface;
 
 class MagazineExtensionRuntime implements RuntimeExtensionInterface
 {
-    public function __construct(private readonly Security $security)
+    public function __construct(
+        private readonly Security $security,
+        MagazineSubscriptionRepository $magazineSubscriptionRepository)
```

Comment:
this should be `private readonly` as well, shouldn't it?
Then the `$this->magazineSubscriptionRepository = $magazineSubscriptionRepository;` line can be deleted as well

Reply:
Sure, done